types/telegram-web-app/index.d.ts

```diff
@@ -0,0 +1,290 @@
+// Type definitions for non-npm package telegram-web-app 1.0
+// Project: https://telegram.org/js/telegram-web-app.js
+// Definitions by: KnorpelSenf <https://github.com/KnorpelSenf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare var Telegram: Telegram;
```

Comment:
```suggestion

declare var Telegram: Telegram;
```
(minor stylistic update)